samples/createHttpTask.js

```diff
@@ -51,6 +51,9 @@ function main(
 
     const task = {
       httpRequest: {
+        headers: {
+          'Content-Type': 'application/json',
```

Comment:
This doesn't match the sample. The sample checks down below if there is even a payload. The example payload is a string or 'text/plain' not json.

Reply:
Ty for catching this!